### ISSUE TITLE
Ginkgo CI: build timeout when trying to get pods

### ISSUE THREAD

**Issue by ianvernon** [cilium/cilium#2421]
Relevant output:
```
16:28:19 ------------------------------
16:28:19 K8sTunnelTest 
16:28:19   Check Geneve mode
16:28:19   /root/jenkins/workspace/Ginkgo-CI-Tests-Pipeline/src/github.com/cilium/cilium/test/k8sT/Tunnels.go:94
17:52:57 
17:52:57 • Failure [5078.349 seconds]
17:52:57 K8sTunnelTest
17:52:57 /root/jenkins/workspace/Ginkgo-CI-Tests-Pipeline/src/github.com/cilium/cilium/test/k8sT/Tunnels.go:28
17:52:57   Check Geneve mode [It]
17:52:57   /root/jenkins/workspace/Ginkgo-CI-Tests-Pipeline/src/github.com/cilium/cilium/test/k8sT/Tunnels.go:94
17:52:57 
17:52:57   Expected
17:52:57       <*errors.errorString | 0xc420964a60>: {
17:52:57           s: "Timeout reached: could not get Pods",
17:52:57       }
17:52:57   to be nil
17:52:57 
17:52:57   /root/jenkins/workspace/Ginkgo-CI-Tests-Pipeline/src/github.com/cilium/cilium/test/k8sT/Tunnels.go:98
17:52:57 ------------------------------
17:52:57 SSSSSSSSSSSSSSSSSSSS
17:52:57 ------------------------------
17:52:57 K8sServicesTest 
17:52:57   Check Service
17:52:57   /root/jenkins/workspace/Ginkgo-CI-Tests-Pipeline/src/github.com/cilium/cilium/test/k8sT/Services.go:90
18:02:02 Sending interrupt signal to process
18:02:02 Sending interrupt signal to process
18:02:10 
18:02:10 ---------------------------------------------------------
18:02:10 Received interrupt.  Running AfterSuite...
```

Build link: https://jenkins.cilium.io/job/Ginkgo-CI-Tests-Pipeline/736/execution/node/58/log/
Similar failure: https://jenkins.cilium.io/job/Ginkgo-CI-Tests-Pipeline/719/execution/node/58/log/

Note the large jump in timestamp from `16:28:19` to `17:52:57`. There should not be such a large time differential. This indicates a bug in the timeout code for the Ginkgo CI.

**Comment by eloycoto** [comment]
Issue related with #2477

**Comment by eloycoto** [comment]
Close this issue, it's already fixed in #2477
